FlowCrypt/Controllers/Compose/ComposeViewController.swift

```diff
@@ -21,7 +21,6 @@ private struct ComposedDraft: Equatable {
  * - User can be redirected here from *InboxViewController* by tapping on *+*
  * - Or from *ThreadDetailsViewController* controller by tapping on *reply*
  **/
-@MainActor
```

Comment:
Since `TableNodeViewController` is annotated with `@MainActor` we have it on each inherited type.